worker/peergrouper/desired_test.go

```diff
@@ -40,13 +40,75 @@ type desiredPeerGroupTest struct {
 	expectErr     string
 }
 
+// Member holds configuration information for a replica set member.
```

Comment:
// TestMember mirrors replicaset.Member but simplifies the structure so that test assertions are easier to understand.